script.js

```diff
@@ -0,0 +1,32 @@
+const MENU_OPEN_BUTTON = document.querySelector('.open');
+const MENU_CLOSE_BUTTON = document.querySelector('.close');
+const SIDE_NAV = document.querySelector('.mobile-display');
+const BUTTON_FEATURES = document.querySelector('.features-button');
+const BUTTON_COMPANY = document.querySelector('.company-button');
+const SUB_OPTIONS_FEATURES = document.querySelector('.features');
+const SUB_OPTIONS_COMPANY = document.querySelector('.company')
+
+
+
+
+MENU_OPEN_BUTTON.addEventListener('click', menuController);
+MENU_CLOSE_BUTTON.addEventListener('click', menuController);
+
+BUTTON_FEATURES.addEventListener('click', featuresButtonController);
+BUTTON_COMPANY.addEventListener('click', companyButtonController);
```

Comment:
Move into function "onInit" and call at the end of the JS file

```suggestion
function() onInit() {
    MENU_OPEN_BUTTON.addEventListener('click', menuController);
    MENU_CLOSE_BUTTON.addEventListener('click', menuController);

    BUTTON_FEATURES.addEventListener('click', featuresButtonController);
    BUTTON_COMPANY.addEventListener('click', companyButtonController);
}
```